precompiles/assets/IAssets.sol

```diff
@@ -4,46 +4,75 @@ pragma solidity >=0.8.17;
-/// @dev deposit the client chain assets for the staker,
-/// that will change the state in deposit module
-/// Note that this address cannot be a module account.
-/// @param clientChainLzID The LzID of client chain
-/// @param assetsAddress The client chain asset address
-/// @param stakerAddress The staker address
-/// @param opAmount The amount to deposit
+
+    /// TRANSACTIONS
+    /// @dev deposit the client chain assets for the staker,
+    /// that will change the state in deposit module
+    /// Note that this address cannot be a module account.
+    /// @param clientChainID is the layerZero chainID if it is supported.
+    //  It might be allocated by Exocore when the client chain isn't supported
+    //  by layerZero
+    /// @param assetsAddress The client chain asset address
+    /// @param stakerAddress The staker address
+    /// @param opAmount The amount to deposit
     function depositTo(
-        uint32 clientChainLzID,
+        uint32 clientChainID,
         bytes memory assetsAddress,
         bytes memory stakerAddress,
-        uint256 opAmount
-    ) external returns (bool success, uint256 latestAssetState);
+        uint256 opAmount) external
+    returns (bool success, uint256 latestAssetState);
 
-/// TRANSACTIONS
-/// @dev withdraw To the staker, that will change the state in withdraw module
-/// Note that this address cannot be a module account.
-/// @param clientChainLzID The LzID of client chain
-/// @param assetsAddress The client chain asset Address
-/// @param withdrawAddress The withdraw address
-/// @param opAmount The withdraw amount
+    /// TRANSACTIONS
+    /// @dev withdraw To the staker, that will change the state in withdraw module
+    /// Note that this address cannot be a module account.
+    /// @param clientChainID is the layerZero chainID if it is supported.
+    //  It might be allocated by Exocore when the client chain isn't supported
+    //  by layerZero
+    /// @param assetsAddress The client chain asset Address
+    /// @param withdrawAddress The withdraw address
+    /// @param opAmount The withdraw amount
     function withdrawPrincipal(
-        uint32 clientChainLzID,
+        uint32 clientChainID,
         bytes memory assetsAddress,
         bytes memory withdrawAddress,
         uint256 opAmount
     ) external returns (bool success, uint256 latestAssetState);
 
-/// QUERIES
-/// @dev Returns the chain indices of the client chains.
+    /// QUERIES
+    /// @dev Returns the chain indices of the client chains.
     function getClientChains() external view returns (bool, uint32[] memory);
-}
 
+    /// TRANSACTIONS
+    /// @dev register some client chain to allow token registration from that chain, staking
+    /// from that chain, and other operations from that chain.
+    /// @param clientChainID is the layerZero chainID if it is supported.
+    //  It might be allocated by Exocore when the client chain isn't supported
+    //  by layerZero
+    function registerClientChain(
```

Comment:
These should be added to `IsTransaction`

Reply:
Yeah, thanks